views/mail/_conversation_sidebar.php

```diff
@@ -21,7 +21,7 @@
             <strong><span class="visible-xs-inline"><?=Icon::get('bars')?></span> <?= Yii::t('MailModule.views_mail_index', 'Conversations') ?></strong>
         </a>
         <?php if($canStartConversation) : ?>
-            <?= NewMessageButton::widget(['label' => Yii::t('MailModule.base', '+ Message'), 'right' => true, 'icon' => false, 'cssClass' => 'hidden-xs'])?>
+            <?= NewMessageButton::widget(['label' => Icon::get('envelope')->tooltip('+ Message'), 'right' => true, 'icon' => false, 'cssClass' => 'hidden-xs'])?>
```

Comment:
Here I'd just use a button with `+` without icon, what do you think, and a tooltip like `Yii::t('MailModule.base', 'Create Conversation')`

Reply:
Seeing as just a `+` is already used in the conversation sidebar it wouldn't hurt doing this here as well

Reply:
### Examples
#### Mobile view
![Screenshot_4](https://user-images.githubusercontent.com/35392110/102144142-fe42a700-3e32-11eb-98a5-408f5dacdd7e.png)
#### Desktop view
![Screenshot_5](https://user-images.githubusercontent.com/35392110/102144148-013d9780-3e33-11eb-99a3-68414425f6f4.png)

Reply:
I think we should keep the `+` since its used in other modules to indicate "add xy" features. Also we have duplicate mail icons in this vew a simple `+` is more descriptive than a envelop icon in my opinion.

Reply:
Then in this case we could use something like this instead?
```php
<?= NewMessageButton::widget(['label' => Icon::get('plus')->tooltip(Yii::t('MailModule.base', 'Create Conversation')), 'right' => true, 'icon' => false, 'cssClass' => 'hidden-xs'])?>
```

As I'm looking though the code this is the only way that I'm seeing that we can implement tooltips without using bootstrap tooltips and it's still the same desired affect :thinking:

Or do you mean use `+` for both the dropdown & conversation sidebar? Either would work and would probably be for the best anyway

Reply:
I'd just change the button used in user profiles and for now leave the `+ Message`.